pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1548,6 +1581,7 @@ func (n *NetworkPolicyController) deleteInternalNetworkPolicy(name string) {
 	internalNetworkPolicy := obj.(*antreatypes.NetworkPolicy)
 	n.internalNetworkPolicyStore.Delete(internalNetworkPolicy.Name)
 	n.cleanupOrphanGroups(internalNetworkPolicy)
+	n.labelidentityInterface.DeletePolicySelectors(internalNetworkPolicy.Name)
```

Comment:
Do we need to check internalNP.SourceRef.Type before calling this? `deleteInternalNetworkPolicy` will be called by three types of NetworkPolicy. I feel there is no need to call this line if it's `controlplane.K8sNetworkPolicy`

Reply:
Thanks for the suggestion. Updated